app/assets/javascripts/autocomplete_checkboxes.js

```diff
@@ -0,0 +1,90 @@
+function hitEnter(event, element, formType, inputType) {
+  if (event.which === 13) {
+    enterOnAutocomplete(element, formType, inputType);
```

Comment:
the input field has `Press ENTER to add`, but I can't open the quick create window unless I typed a single character. Would this behaviour seem confusing to some users?